Description: Set default job error policy to "retry-job", since it is less
 confusing and a better default on most machines.
 .
 Amend documentation accordingly.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Last-Update: 2016-07-03

--- a/scheduler/conf.c
+++ b/scheduler/conf.c
@@ -787,7 +787,7 @@
   cupsdSetString(&LPDConfigFile, CUPS_DEFAULT_LPD_CONFIG_FILE);
   cupsdSetString(&SMBConfigFile, CUPS_DEFAULT_SMB_CONFIG_FILE);
 
-  cupsdSetString(&ErrorPolicy, "stop-printer");
+  cupsdSetString(&ErrorPolicy, "retry-job");
 
   JobHistory          = DEFAULT_HISTORY;
   JobFiles            = DEFAULT_FILES;
@@ -1309,8 +1309,8 @@
       strcmp(ErrorPolicy, "retry-job") &&
       strcmp(ErrorPolicy, "stop-printer"))
   {
-    cupsdLogMessage(CUPSD_LOG_ALERT, "Invalid ErrorPolicy \"%s\", resetting to \"stop-printer\".", ErrorPolicy);
-    cupsdSetString(&ErrorPolicy, "stop-printer");
+    cupsdLogMessage(CUPSD_LOG_ALERT, "Invalid ErrorPolicy \"%s\", resetting to \"retry-job\".", ErrorPolicy);
+    cupsdSetString(&ErrorPolicy, "retry-job");
   }
 
  /*
